docs/src/index.md

```diff
@@ -2,12 +2,15 @@
 
 [Cclib.jl](https://github.com/cclib/Cclib.jl) is a Julia wrapper around [Cclib](https://cclib.github.io/index.html) - an open source library written in Python for parsing and interpreting the results of computational chemistry packages.
 
-It allows to perform natively in Julia the following:
+# Features
 
-- Parse data from the output files generated by computational chemistry programs, such as Gaussian, GAMESS, PSI4 etc.
-- Provide a consistent interface to the results of computational chemistry calculations, particularly those results that are useful for algorithms or visualization
-- Facilitate the implementation of algorithms that are not specific to a particular computational chemistry package
-- Maximize interoperability with other open source computational chemistry and cheminformatic software libraries
+- Parsing outputs of quantum chemistry programs, such as such as Gaussian, GAMESS, and PSI4.
```

Comment:
can you grab the list of programs from https://github.com/cclib/cclib/blob/b8d42a163ce9bafd4b660e2a933f56a8cc54fd9b/ANNOUNCE#L3